### DISCUSSION TITLE
Kôika HDL

### DISCUSSION THREAD

**Discussion by omasanori** [B-Lang-org/bsc#423] [General]
I am afraid that this is off-topic, but I would share [Kôika](https://github.com/mit-plv/koika), a rule-based HDL influenced by Bluespec HDL embedded in Coq. Some ideas developed for Kôika could improve Bluespec HDL too.

**Comment by quark17** [comment]
Thanks!  It's also mentioned in Discussion #333 which lists various Formal Verification directions.
